### DISCUSSION TITLE
What happen to "existsAt" in Webpack 5

### DISCUSSION THREAD

**Discussion by rogerprz** [webpack/webpack#15304] [Q&A]
# Bug report

**What is the current behavior?**
Missing compilation and "existsAt"

**If the current behavior is a bug, please provide the steps to reproduce.**
I am not sure if this is a bug and can be similar to https://github.com/webpack/webpack/issues/12313
I am upgrading my teams codebase to use webpack 5 and I no longer have access to Compilations in stats.
The reason I need this path is because I use it for the output path which I cannot find in webpack 5. 
```
function printStats(stat) {
    if (stat.compilation && stat.compilation.assets) { // <-- compilation doesn't exist
        for (const asset in stat.compilation.assets) {
            const assetInfo = stat.compilation.assets[asset];
            const assetPath = path.relative(process.cwd(), assetInfo.existsAt); 
            console.log(`Output file "${chalk.cyan(assetPath)}"`);
        }
    }
}
```

**What is the expected behavior?**
Expected behavior is for me to be able to find compilation and existsAt

**Other relevant information:**
webpack version:  "webpack": "^5.68.0",
Node.js version: 14.x
Operating System: MacOs
Additional tools:

**Comment by vankop** [comment]
hm.. strange.. for asset stats you can use `stat.toJson({ assets: true })`
